packages/admin/admin/src/statusBadge/StatusBadge.styles.ts

```diff
@@ -0,0 +1,22 @@
+import { Theme } from "@mui/material";
+import { createStyles } from "@mui/styles";
```

Comment:
I think it's fine for now. 
The Theming-Refactor is currently only done in a feature-branch: https://github.com/vivid-planet/comet/pull/1376
It will be refactored there eventually anyway.